content.md

```diff
@@ -1735,36 +1782,39 @@ damage they can, and the unmended damage leaves them wounded.
-damage inflicted by a Carrion Crows) or a non-minion source (such as the
-damage inflicted by a retainer).
-
-Note that damage that is not inflicted by a strike (such as environmental 
-damage from Carrion Crows or Murder of Crows) cannot be dodged, since 
-dodging only protects from the opponent's strike.
-
-**Aggravated damage:** Some damage is described as aggravated damage,
-such as damage from fire, sunlight, or from the claws or fangs of some
-other supernatural creatures, including some vampires. Aggravated damage
-differs from normal damage in two ways: aggravated damage cannot be
-mended, and aggravated damage can burn a vampire if that vampire is
-already wounded. Since it cannot be mended, the vampire does not burn
-any blood to mend it, but instead becomes wounded (goes straight to
-torpor), unless the damage has been prevented.
-
-Aggravated damage done to a wounded vampire can burn the vampire
-outright. A wounded vampire is one that has taken damage that they
-failed to mend or one that is in torpor or on their way to torpor.
+> **DIFFERENT KINDS OF DAMAGE**\
+> \
+> **Normal damage**: Any damage which is not specified otherwise.\
+>\
+> **Environmental damage:** Any damage not inflicted by a minion either as
+> a strike, or by an explicit effect ("this minion inflicts...") is
+> environmental. Environmental damage can have no source (such as the
+> damage inflicted by a Carrion Crows) or a non-minion source (such as the
+> damage inflicted by a retainer).\
+>\
+> Note that damage that is not inflicted by a strike (such as environmental
+> damage from Carrion Crows or Murder of Crows) cannot be dodged, since
+> dodging only protects from the opponent's strike.\
+> \
+> **Aggravated damage:** Some damage is described as aggravated damage,
+> such as damage from fire, sunlight, or from the claws or fangs of some
+> other supernatural creatures, including some vampires. Aggravated damage
+> differs from normal damage in two ways: aggravated damage cannot be
+> mended, and aggravated damage can burn a vampire if that vampire is
+> already wounded. Since it cannot be mended, the vampire does not burn
+> any blood to mend it, but instead becomes wounded (goes straight to
+> torpor), unless the damage has been prevented.\
+> \
+> Aggravated damage done to a wounded vampire can burn the vampire outright. A wounded vampire is one that has taken damage that they
+> failed to mend or one that is in torpor or on their way to torpor.
 
 For each point of aggravated damage that is successfully inflicted on a
 wounded vampire, they must burn 1 blood to prevent their destruction. If
 they do not have enough blood, they are burned. Destruction by this
 method does not constitute diablerie
 (see [**Diablerie**](https://www.vekn.net/rulebook#diablerie)).
 
-If both regular damage and aggravated damage are successfully inflicted
+If both normal damage and aggravated damage are successfully inflicted
```

Comment:
now you hit a conundrum because your new presentation begs the question: "What about environmental damage"? Since you've separated them from "Normal" damages in the sections just before. You might want to keep instead `regular damage (normal or environmental)`, or maybe switch simply to `non-aggravated damage` (as on the cards texts)

Reply:
if card texts say non-aggravated, imo it would be best to stick with that :)